Scripts/Core/EditorVR.cs

```diff
@@ -190,7 +190,7 @@ void Awake()
 			UnityBrandColorScheme.sessionGradient = UnityBrandColorScheme.GetRandomGradient();
 
 			var sceneObjectModule = AddModule<SceneObjectModule>();
-			sceneObjectModule.shouldPlaceObject = (obj, targetScale) =>
+			sceneObjectModule.tryPlaceObject = (obj, targetScale) =>
```

Comment:
I'm not sure I agree with the rename here semantically. When you look at how it is used in SceneObjectModule, the module is asking whether it should place the object. Now when I read this, it is like TryGetValue where I'm expecting the delegate to try and place the object now. If you want to keep the naming, then we'll need to reverse the logic of where it is used to `if (tryPlaceObject == null || !tryPlaceObject(obj, targetScale))`